.github/workflows/alert_center.yml

```diff
@@ -0,0 +1,66 @@
+name: CI for alert center
+
+on:
+  push:
+    branches: [ "main" ]
+    paths:
+      - 'typescript/alert-center/**'
+      - '.github/workflows/alert_center.yml'
+  pull_request:
+    types:
+      - ready_for_review
+      - opened
+      - reopened
+      - synchronize
+    paths:
+      - 'typescript/alert-center/**'
+      - '.github/workflows/alert_center.yml'
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        node-version: ['20', '22', '23']
```

Comment:
Not sure if we need to test demo apps on all node versions... This may make tests too heavy.

Reply:
In this case it did surface some issue with the `knexfiles` -- happy to remove one (possibly 23 because it is not yet the LTS) if our gha bill is too high or they take too long?